### PR TITLE
replace ginkgo with go test

### PR DESCRIPTION
**Pull request by gheorghestrimtu**

### CODE REVIEW

File: tests/e2e/common/test_common.go [hunk 1]
```diff
@@ -3,16 +3,17 @@ package common
 //revive:disable:dot-imports
```
Comment (kalverra) [inline tests/e2e/common/test_common.go]: Probably best to remove these now that we no longer import Ginkgo

File: tests/e2e/common/test_common.go [hunk 2]
```diff
@@ -208,63 +211,62 @@ func (m *OCRv2TestState) initializeNodesInContractsMap() {
...
 
 	m.initializeNodesInContractsMap()
 	g := errgroup.Group{}
 	for i := 0; i < len(m.ContractsNodeSetup); i++ {
 		i := i
 		g.Go(func() error {
-			defer ginkgo.GinkgoRecover()
 			cd, err := solclient.NewContractDeployer(m.c, m.Env, m.LinkToken)
-			Expect(err).ShouldNot(HaveOccurred())
+			require.NoError(m.T, err)
```
Comment (kalverra) [inline tests/e2e/common/test_common.go:232]: Doesn't have to be done in this PR, but error messages would likely be helpful in some cases.

**Review by kalverra — CHANGES_REQUESTED** [review]

**Review by kalverra — DISMISSED** [review]

File: contracts/programs/access-controller/src/lib.rs [hunk 3]
```diff
@@ -4,7 +4,7 @@ use std::mem;
 
 use arrayvec::arrayvec;
 
-declare_id!("9xi644bRR8birboDGdTiwBq3C7VEeR7VuamRYYXCubUW");
+declare_id!("2ckhep7Mvy1dExenBqpcdevhRu7CLuuctMcx7G9mWEvo");
```
Comment (tateexon) [inline contracts/programs/access-controller/src/lib.rs]: Did you mean to change this?

File: contracts/programs/ocr2/src/lib.rs [hunk 4]
```diff
@@ -4,7 +4,7 @@ use anchor_spl::token;
 use arrayref::{array_ref, array_refs};
 use state::{Billing, Proposal, ProposedOracle, STATE_VERSION};
 
-declare_id!("cjg3oHmg9uuPsP8D6g29NWvhySJkdYdAo9D25PRbKXJ");
+declare_id!("E3j24rx12SyVsG6quKuZPbQqZPkhAUCh8Uek4XrKYD2x");
```
Comment (tateexon) [inline contracts/programs/ocr2/src/lib.rs]: Did you mean to change this?

File: contracts/programs/store/src/lib.rs [hunk 5]
```diff
@@ -7,7 +7,7 @@ mod state;
 use crate::state::with_store;
 pub use crate::state::{NewTransmission, Store as State, Transmission, Transmissions};
 
-declare_id!("HEvSKofvBgfaexv23kMabbYqxasxU3mQ4ibBMEmJWHny");
+declare_id!("9kRNTZmoZSiTBuXC62dzK9E7gC7huYgcmRRhYv3i4osC");
```
Comment (tateexon) [inline contracts/programs/store/src/lib.rs]: Did you mean to change this?

File: contracts/programs/access-controller/src/lib.rs (hunk 3 above)
Comment (gheorghestrimtu) [reply contracts/programs/access-controller/src/lib.rs]: it was changed when doing `anchor build` I can change it back if it's necessary

Comment (tateexon) [reply contracts/programs/access-controller/src/lib.rs]: It is likely the cause of your anchor test hangs in ci.

Comment (gheorghestrimtu) [reply contracts/programs/access-controller/src/lib.rs]: ohhh ok, I'll reverse

**Review by kalverra — APPROVED** [review]

**Review by tateexon — APPROVED** [review]

**Pull request merged**